holoviews/plotting/plot.py

```diff
@@ -145,8 +145,9 @@ class DimensionedPlot(Plot):
     show_title = param.Boolean(default=True, doc="""
         Whether to display the plot title.""")
 
-    title_format = param.String(default="{label} {group}", doc="""
-        The formatting string for the title of this plot.""")
+    title_format = param.String(default="{label} {group}\n{dimensions}", doc="""
+        The formatting string for the title of this plot, allows defining
+        a label group separator and dimension labels.""")
```

Comment:
Ok. That means there will be quite a lot of test data to update. We need to be careful about transient glitches and need to review the display output thoroughly.